eosdactoken/eosdactoken.abi

```diff
@@ -1,134 +1,193 @@
+	}, {
+		"name": "memberreg",
+		"type": "membereg",
+		"ricardian_contract": "## ACTION: memberreg\n**PARAMETERS:** __sender__ is a type of eosio account_name, __agreedterms__ is a hash reference to a document contained in a string with a maximum of 256 charaters\n**INTENT:** The intent of memberreg is to indicate that the account has agreed to the terms of the DAC. It will update an internal database of member accounts. This action must supply the hash of the agreement in {{ agreedterms }}, it will hold the most recently agreed to, and can be called multiple times to update the hash.\n**TERM:** This action lasts for the duration of the processing of the contract. The member registration will persist on the deployed contract as long as it is active or superceeded by an updated memberreg or memberunreg action."
+	}, {
+		"name": "memberunreg",
+		"type": "memberunreg",
+		"ricardian_contract": "## ACTION: memberunreg\n**PARAMETERS:** __sender__ is a type of eosio account_name\n**INTENT:** The intent of memberunreg is to allow an account {{ sender }} to unregister it's membership.\n**TERM:** This action lasts for the duration of the processing of the contract. The action will persist on the deployed contract as long as it is active or superceeded by an updated memberreg action."
+	}],
+	"tables": [{
+		"name": "accounts",
+		"type": "account",
+		"index_type": "i64",
+		"key_names": ["currency"],
+		"key_types": ["uint64"]
+	}, {
+		"name": "stat",
+		"type": "currency_stats",
+		"index_type": "i64",
+		"key_names": ["currency"],
+		"key_types": ["uint64"]
+	}, {
+		"name": "members",
+		"type": "member",
+		"index_type": "i64",
+		"key_names": ["sender"],
+		"key_types": ["uint64"]
+	}],
+	"ricardian_clauses": [{
+			"entire_agreement": "## ENTIRE AGREEMENT.\nThis contract contains the entire agreement of the parties, for all described actions, and there are no other promises or conditions in any other agreement whether oral or written concerning the subject matter of this Contract. This contract supersedes any prior written or oral agreements between the parties. "
+		},
+		{
+			"binding_constitution": "## BINDING CONSTITUTION:\nAll the the action descibed in this contract are subject to the EOSDAC consitution as held at http://eosdac.io . This includes, but is not limited to membership terms and condiutions, dispute resolution and severability."
```

Comment:
And finally this one. 
Please double check this if you have not already for legal stuff.